resources/server/sv_twitter.ts

```diff
@@ -132,8 +133,12 @@ async function createTweet(identifier: string, tweet: NewTweet): Promise<Tweet>
     tweet.images,
     tweet.retweet,
   ]);
+  // This should not be an any type and instead should be
+  // a Tweet[] according to mysql2 documentation. But instead
```

Comment:
Are you sure on this? I am not sure this is the case.

`console.log(Object.keys(insertData));` is returning:

```
[
  "fieldCount",
  "affectedRows",
  "insertId",
  "info",
  "serverStatus",
  "warningStatus"
]
```

And this is the return value of the `await pool.query....`:

```
cfx> [
  {
    "fieldCount": 0,
    "affectedRows": 1,
    "insertId": 33,
    "info": "",
    "serverStatus": 2,
    "warningStatus": 0
  },
  null
]
```
and this is function is working as expected on latest master.

Reply:
Was a little out of it when I wrote that apparently as double checking now, we should use this type
```ts
declare interface ResultSetHeader {
    constructor: {
        name: 'ResultSetHeader'
    };
    affectedRows: number;
    fieldCount: number;
    info: string;
    insertId: number;
    serverStatus: number;
    warningStatus: number;
}
```

Reply:
> and this is function is working as expected on latest master.

There is no change in functionality with this PR, just an added comment explaining the any

Reply:
Fixed in 357acd4407b1368df55fd5a26a56dc849b8b8ec9 

![](http://i.tasoagc.dev/MkQx)